x/campaign/types/params.go

```diff
@@ -86,3 +100,10 @@ func validateCampaignCreationFee(i interface{}) error {
 	}
 	return v.Validate()
 }
+
+func validateMaxMetadataLength(i interface{}) error {
```

Comment:
Can't see a test for this method, do we have one?

Reply:
added